sample-apps/springboot/build.gradle.kts

```diff
@@ -4,6 +4,10 @@ plugins {
   id("com.google.cloud.tools.jib")
 }
 
+
+//This property override Spring Framework to the version that contains the fix
+ext['spring-framework.version'] = '5.3.18'
```

Comment:
We don't need this if we have updated to spring-boot `v2.6.6`, do we?  I'd not want to see this become stale unnecessarily.

Reply:
I am not 100% sure, but for building the `sample-app`
image here. it is not clear for me either which version of `spring-framework` it is built off. overriding to the fixed version should be okay I guess?. 

Also from [info](https://spring.io/blog/2022/03/31/spring-framework-rce-early-announcement) and [example](https://github.com/snicoll-scratches/spring-boot-cve-2022-22965/blob/main/spring-boot-2.4-gradle/build.gradle#L16-L18)
 
<img width="776" alt="image" src="https://user-images.githubusercontent.com/41936996/161345455-c3d82a9e-31b2-4ce4-b1a9-6187f288bb0b.png">

Reply:
I am not 100% sure, but for building the `sample-app`
image here. it is not clear for me either which version of `spring-framework` it is built off. overriding to the fixed version should be okay I guess?. 

Also from [info](https://spring.io/blog/2022/03/31/spring-framework-rce-early-announcement) and [example](https://github.com/snicoll-scratches/spring-boot-cve-2022-22965/blob/main/spring-boot-2.4-gradle/build.gradle#L16-L18)
 
<img width="776" alt="image" src="https://user-images.githubusercontent.com/41936996/161345455-c3d82a9e-31b2-4ce4-b1a9-6187f288bb0b.png">

Reply:
That makes sense. I thought since I couldn't explicitly mention the version, I'd add the code to override the Spring version